socorro/unittest/external/es/base.py

```diff
@@ -167,7 +167,9 @@
         'is_mandatory': False,
         'storage_mapping': {
             'type': 'date',
-            'format': 'yyyy-MM-dd\'T\'HH:mm:ssZZ||yyyy-MM-dd\'T\'HH:mm:ss.SSSSSSZZ'
+            'format': (
+                'yyyy-MM-dd\'T\'HH:mm:ssZZ||yyyy-MM-dd\'T\'HH:mm:ss.SSSSSSZZ'
+            )
```

Comment:
I changed this because it was the only thing missing from making this file pep8 compliant and git hooks kept nagging me.